backend/functions/src/emailing/routes.ts

```diff
@@ -49,4 +48,28 @@ router.post('/send', (req, res) => {
     )
 })
 
+/** @.com/api root/email/timestamp
+ *
+ * @param courseId = string of 6-digit course code
+ * @param groups = string [] of groups to be sent emails
+ *
+ * @yeilds email sent timestamp update in database
```

Comment:
yeilds

Reply:
Yeab good points. I don't really know tbh. Do we need store timestamps for other types of emails sent? I was thinking that we needed this so the front-end & LSC Staff know which groups they sent emails to or something. If there aren't that many types of emails to sent (matched, check-ins, adding students, asking to join, etc.) then maybe make the timestamp field an enum or something, and then pass that into the function that updates the field according to the enum type? Kind of raw and brute force method though. Or, just have separate functions for each type of shared email if we don't have a lot.

Not 100% sure on the best code design as not 100% sure when/how/what scenarios we would use it and for what purpose. I'll try to think more about it once I finish the email modal functionality.

Reply:
Yeah maybe some sort of mapping between timestamp-type string given to the function and what property in the database it should update could work. I'll think more about it too